web/impact/impact/v1/helpers/criterion_option_spec_helper.py

```diff
@@ -44,6 +45,33 @@ class CriterionOptionSpecHelper(ModelHelper):
     ]
     INPUT_KEYS = ALL_KEYS
 
+    def __init__(self, subject):
+        super().__init__(subject)
+        criterion = self.subject.criterion
+        self.criterion_helper = CriterionHelper.find_helper(
+            criterion.type, criterion.name)(subject)
+
+    def app_ids_for_feedback(self, feedbacks, option_name, applications):
```

Comment:
it is unfortunate that we use feedback both as a mass noun and as a count noun in the same declaration. Could this be `app_ids_for_feedbacks`? Or `feedback_set` in both places?

Reply:
Reasonable.  Switched to app_ids_for_feedbacks.